lms/djangoapps/bulk_user_retirement/views.py

```diff
@@ -57,6 +57,8 @@ def post(self, request, **kwargs):  # pylint: disable=unused-argument
                 user_to_retire = User.objects.get(username=username)
                 with transaction.atomic():
                     create_retirement_request_and_deactivate_account(user_to_retire)
+                log.info(f'The user "{username}" has been added to the retirement pipeline \
+                         by "{request.user}"')
```

Comment:
Why is this needed? Also, the commit type should be chore/temp not fix.

Reply:
The ticket mentioned that it would be nice to log the support user triggering the retirement flow. Also I have fixed the commit type. Turns out google is not the best place for suggestions regarding commit type :-)

Reply:
You can read about conventional commits in details https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html